static/robots.txt

```diff
@@ -1 +1,5 @@
 Sitemap: https://pulsar.apache.org/sitemap.xml
+User-agent: *
+Allow: /
+Disallow: /api/
```

Comment:
We should exclude legacy versions only. Maintained versions should be still indexed.